packages/help-center/src/components/help-center-support-chat-message.scss

```diff
@@ -1,4 +1,7 @@
 .help-center-support-chat__conversation-container {
+	&:first-child {
```

Comment:
imo it would be better if we just remove the border-bottom: none [here](https://github.com/Automattic/wp-calypso/blob/2a1318ccd299bc565e8794fc3fb534d319f74377/packages/help-center/src/components/help-center-header.scss#L17)

Reply:
100% better 
Thank you!